.devcontainer/.env

```diff
@@ -7,6 +7,15 @@ S3_BUCKET_NAME=resumes
 S3_ROOT_ACCESS_KEY=minioadmin
 S3_ROOT_SECRET_KEY=minioadmin
 
+DATABASE_URL=postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}/${POSTGRES_DATABASE}?schema=public&connection_limit=10&pgbouncer=true
+SHADOW_DATABASE_URL=postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}/${POSTGRES_DATABASE}?schema=public
+
+DISCORD_TOKEN="MTE2NDY2NDkxMjI3MDg3MjY2Nw.Gb5Byz.LTMz54pcCIBvlUemNinMvN1rwCN85EQACqLcpg"
```

Comment:
These discord environment variables should _not_ be committed. Please use the .env.local file instead for these (and put a reference to it in .env.local.default

Also, you will want to invalidate/regenerate this token in your Discord Developer Portal.